src/EditorFeatures/CSharpTest/ReplaceDefaultLiteral/ReplaceDefaultLiteralTests.cs

```diff
@@ -0,0 +1,856 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.ReplaceDefaultLiteral;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.Diagnostics;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.ReplaceDefaultLiteral
```

Comment:
small nit.  The name is a bit vague to me here.  I might consider it ReplaceDefaultLiteralInCase.  But it's very minor.  if you like this name, i'm totally ok with it.

Reply:
In the future, this should also be extended to support replacing usage of default literals in C# 7.0. AFAIK the only change necessary would be adding additional diagnostic ids, so I don't think that would warrant a separate code fix. "ReplaceDefaultLiteralInCase" would be not be an appropriate name then.

Reply:
The purpose of this fix is essentially: replace a default literal with something else if it's illegal here.